### ISSUE TITLE
epsilon and reg_lambda not defined in the inotebook version

### ISSUE THREAD

**Issue by turgu1** [dennybritz/nn-from-scratch#4]
Adding the two variable definitions from the web page and putting it just before the calculate_loss function did the trick.

**Comment by dennybritz** [comment]
Good catch. Do you mind submitting a PR with the working version?

**Comment by turgu1** [comment]
I tried to do it but doing the modification in Jupiter creates a lot of other changes in the file beyond adding the three lines that follow. As I'm using Python v3.x, I don't wan't to generate more issues...:

# Gradient descent parameters (I picked these by hand)
epsilon = 0.01 # learning rate for gradient descent
reg_lambda = 0.01 # regularization strength

**Comment by jtpio** [comment]
All the other changes are probably related to the cell metadata used for the notebook (collapsed, execution_count...).
Anyway, #5 should fix it! (and the diff is kept to the bare minimum).

**Comment by dennybritz** [comment]
Thanks! Yeah, the other changes are the probably the cached results in the notebook. Could also just clean the output after changing the code. Anyway, I'm closing this. Thanks for the PR.
